test/models/rental_test.rb

```diff
@@ -0,0 +1,17 @@
+require "test_helper"
+
+describe Rental do
+
+  describe "relations" do
+    it "must respond to customer" do
```

Comment:
You've done such a good job of encapsulating your logic in the model! One of the big advantages of that is that you can test the logic in isolation here, but you're not doing that.